src/Popover.js

```diff
@@ -34,34 +30,66 @@ const Popover = ({
     dataTest,
     elevation,
     maxWidth,
+    observePopperResize,
+    observeReferenceResize,
     placement,
-    onBackdropClick,
-}) =>
-    createPortal(
-        <Backdrop onClick={onBackdropClick} transparent>
-            <Popper
-                dataTest={`${dataTest}-popper`}
-                placement={placement}
-                reference={reference}
-                modifiers={arrow ? arroModifiers : []}
+    onClickOutside,
+}) => {
+    const referenceElement =
+        reference instanceof Element
+            ? reference
+            : reference && reference.current
```

Comment:
Minor : this can be written `reference?.current` with [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining), which we support in our babel config.  Optional if you prefer `&&` here though.

Reply:
I'd like to start using this syntax but currently this project or my editor doesn't seem to like it. As discussed on Slack, let's fix this later in `dhis2/ui`.